src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -254,12 +263,25 @@ class MetadataBasedItemList extends React.Component<Props, State> {
                 );
                 break;
             default: {
-                const key = isString(metadataColumn) ? metadataColumn : metadataColumn.key;
-                const field = find(fields, ['key', key]);
-                if (!field) {
-                    return cellData;
+                const key = isString(fieldToShow) ? fieldToShow : fieldToShow.key;
+                let field;
+                let type = FIELD_TYPE_STRING;
```

Comment:
Just noticed this after looking at the updates in `MetadataBasedItemList.test.js`. Is `string` the default type because the code replaces `undefined` values with "No Value"?

Reply:
@sahiga the item fields that we are planning to make editable are mostly going to be `name` and `description`. These are of type `string`. But if its a metadata field, then we overwrite the value of `type` in line 279
```({ type, value, options = [] } = field);```